documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -5,26 +5,29 @@
 [id='con-kafka-connect-authentication{context}']
 = Authentication support in Kafka Connect
 
-Authentication is configured through the `authentication` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`.
-The `authentication` property specifies the type of the authentication mechanisms which should be used and additional configuration details depending on the mechanism.
-The supported authentication types are:
+You can configure authentication for Kafka Connect in the `authentication` property in `KafkaConnect.spec` and `KafkaConnectS2I.spec`. 
+The `authentication` property specifies which of the supported authentication mechanisms to use, and additional configuration depending on the mechanism.
+
+The following types of authentication are supported for Kafka Connect:
 
 * TLS client authentication
 * SASL-based authentication using the SCRAM-SHA-512 mechanism
 * SASL-based authentication using the PLAIN mechanism
 * xref:assembly-oauth-authentication_str[OAuth 2.0 token based authentication]
 
-== TLS Client Authentication
+The following sections explain how each authentication mechanism works. 
+
+== TLS Client authentication
 
-To use TLS client authentication, set the `type` property to the value `tls`.
+To use TLS client authentication, set the `authentication.type` property to `tls`.
 TLS client authentication uses a TLS certificate to authenticate.
-The certificate is specified in the `certificateAndKey` property and is always loaded from a Kubernetes secret.
-In the secret, the certificate must be stored in X509 format under two different keys: public and private.
+The certificate is specified in the `certificateAndKey` property and is always loaded from a Kubernetes Secret.
+In the Secret, the certificate must be stored in X509 format under two different keys: public and private.
```

Comment:
Use of "keys" here is ambiguous. And TBH I'm not sure I follow what this even means. A certificate is a data structure which includes a public key and some other information. A certificate does not include a private key. So the sentence doesn't make sense because it says the certificate needs to be stored under a private "key" (aka field or property), but there's nothing private about the cert.

```suggestion
In the Secret, the certificate must be stored in X509 format under the field named by the `certificate` and the private key must be stored under the field named by the `key` property, as shown below.
```